src/app/employer/(home)/[viewId]/page.tsx

```diff
@@ -97,13 +93,14 @@ const JobDetailsPage = ({ params: { viewId } }: { params: { viewId: string } })
         try {
             const response = await axios.put(
```

Comment:
donot harcode the url, extract it to a promise and use it with tanstack/react-query